### PR TITLE
Updates to CIME statistical ensemble test ensemble generation scripts

### PR DESCRIPTION
**Pull request by allibco**
-Removed old *.sh files in favor of the python versions.  
-Updated to allow generation of POP-ECT ensemble.
-Updated README files.
-Updated default case configurations and removed options that are no longer needed.

Test suite: 
Test baseline: 
Test namelist changes: 
Test status:

Fixes [CIME Github issue #] N/A
User interface changes?:  Yes - python script instead of shell script (same options).
Update gh-pages html (Y/N)?: N

Code review:

### CODE REVIEW

File: tools/statistical_ensemble_test/README [hunk 1]
```diff
@@ -1,56 +1,84 @@
...
+	   needs to be larger than the number of variables that ECT will evaluate.
 
-	      (or, replace "ensemble.sh" with "python ensemble.py")
 
-Notes: -case name must end in ".000" and include the full path
-       -ensemble size must be specified, suggested defaults are 151 
-       for CAM-ECT and 350 for UF-CAM-ECT. Note that the ensemble size
-       needs to be larger than the number of variables that ECT will
-       evaluate.
+HERE
```
Comment (jedwards4b) [inline tools/statistical_ensemble_test/README]: Is this intentional?  It seems to be out of place.

File: tools/statistical_ensemble_test/single_run.py [hunk 2]
```diff
@@ -213,11 +231,15 @@ def single_case(opts_dict, case_flags, stat_dir):
     print('STATUS: create_newcase flags = ' + case_flags)
     command = './create_newcase ' + case_flags
     ret = os.system(command)
+    #print(ret)
```
Comment (jedwards4b) [inline tools/statistical_ensemble_test/single_run.py]: remove commented print statement

File: tools/statistical_ensemble_test/single_run.py [hunk 3]
```diff
@@ -234,84 +256,100 @@ def single_case(opts_dict, case_flags, stat_dir):
     ret = os.system(command)
 
     #time steps
-    if opts_dict['uf'] == True:
-        command = './xmlchange --file env_run.xml --id STOP_OPTION --val nsteps'
-        ret = os.system(command)
-        command = '  ./xmlchange --file env_run.xml --id STOP_N --val 9'
-        ret = os.system(command)
+    if opts_dict['ect'] == 'pop':
```
Comment (jedwards4b) [inline tools/statistical_ensemble_test/single_run.py]: We are using python - I think these should be changed from xmlchange commands to directly access the case object.

**Comment by jedwards4b** [comment]
@allibco please also fix the whitespace issues noted here https://app.codacy.com/app/jedwards4b/cime/pullRequest?prid=2391432

File: tools/statistical_ensemble_test/README (hunk 1 above)
Comment (allibco) [reply tools/statistical_ensemble_test/README]: Oops - not intentional

File: tools/statistical_ensemble_test/single_run.py (hunk 2 above)
Comment (allibco) [reply tools/statistical_ensemble_test/single_run.py]: got it

File: tools/statistical_ensemble_test/single_run.py (hunk 3 above)
Comment (allibco) [reply tools/statistical_ensemble_test/single_run.py]: OK - will change in future PR

**Pull request merged**